suite-native/module-add-accounts/src/hooks/useAddCoinAccount.ts

```diff
@@ -317,7 +317,16 @@ export const useAddCoinAccount = () => {
                 screen = AppTabsRoutes.ReceiveStack;
             }
 
-            showGeneralErrorAlert();
+            if (newAccountResult.payload === 'Passphrase is incorrect') {
```

Comment:
I would prefer to match this based on error type rather then string value. But I understand that would mean to refactor all the add new coin error types. So feel free to resolve this.

Reply:
Not add account types, but connect error types. This is from connect, which sends something like:
```
{ code: "Device_InvalidState", error: "Passphrase is incorrect" }
```

I agree this is not ideal, but right now it's our only option without having to change connect code.

Reply:
It would be definitely better to use error code `Device_InvalidState`, not the message...

Reply:
I assume that the invalid state can be tied to multiple other cases that would trigger this alert even if it wasn't passphrase. I plan to check this with someone next week to improve the condition